client/batch.go

```diff
@@ -308,6 +315,29 @@ func (b *Batch) Scan(s, e interface{}, maxRows int64) {
 	b.initResult(1, 0, nil)
 }
 
+// ReverseScan retrieves the rows in reserve order starting at end (exclusive)
+// and finishing at begin(inclusive).
+//
+// A new result will be appended to the batch which will contain up to maxRows
+// rows and Result.Err will indicate success or failure.
+//
+// key can be either a byte slice, a string, a fmt.Stringer or an
+// encoding.BinaryMarshaler.
+func (b *Batch) ReverseScan(s, e interface{}, maxRows int64) {
```

Comment:
Now the semantics of ReverseScan is same with Scan(start key inclusive and end key exclusive) except seeking order.  Is it better to add a reverse flag in proto.ScanRequest instead of adding a new ReverseScanRequest ?